src/arsort.py

```diff
@@ -663,3 +662,24 @@ def make_dirs(argflag, fitsdict, filesort):
     else: os.mkdir(newdir)
     # Return the name of the science targets
     return sci_targs
+
+
+def set_instr_name(inst_name):
```

Comment:
It might be better to have this as a parameter in the settings file. Previously, I had been using (what is now called):
det01 suffix
Perhaps it makes sense to have a variable in the settings file called (something along the lines of):
mosaic shortname LRISb
mosaic name LRISb